### PR TITLE
Add Vantiq Camel Assembly document (readme).

### LINKED ISSUE #444: Add Camel Assemblies doc/readme

### PR DESCRIPTION
**Pull request by fhcarter**
Fixes #444

Adds README.md to the camelAssemblies subproject.

This provides an overview of the contents and its use.

(This is likely to be the final PR into this staging repository.)

### CODE REVIEW

File: camelAssemblies/README.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+The Vantiq Camel assemblies contained herein are composed of the following.
+
+* Vantiq Source -- this is the Vantiq component that represents the external system to the Vantiq environment.
+  * The Vantiq source will be of type `CAMEL`. This is defined in the Camel Connector assembly `com.vantiq.extsrc.camelconn.camelConnector`.
+  * The source's configuration will include the Camel route to/from (source/sink) the external source via Camel 
+    Component(s).
+  * The configuration may include configuration information (such as credentials, type of interaction) generated 
+    from Vantiq assembly configuration parameters.
+  * The Vantiq source is the in the package matching the assembly name, and name consisting of the base name with the
```
Comment (jcguion) [inline camelAssemblies/README.md:67]: typo - is ~the~ in the package ... ?

File: camelAssemblies/README.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+## Using the Vantiq Camel Assemblies
+
+Once installed, each Vantiq Camel assembly creates a source & service for interacting with that source.
+
+The source is defined, but it is implemented via a [Camel Connector](../camelConnector/README.md). To run the Camel 
+Connector, please see the [Camel Connector overview](../camelConnector/README.md).
+
+As noted above, the service will have an inbound or outbound event defined, and interactions with the service are
+generally performed by publishing to that event or handling it (for _inbound_ or _outbound_ events).
+All of the defined to use the _structured messages_ as defined by the
```
Comment (jcguion) [inline camelAssemblies/README.md:122]: typo `All of the defined to use ...`

File: camelAssemblies/README.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+As noted above, the service will have an inbound or outbound event defined, and interactions with the service are
+generally performed by publishing to that event or handling it (for _inbound_ or _outbound_ events).
+All of the defined to use the _structured messages_ as defined by the 
+[Camel Component](../camelComponent/README.md#structured-headers-and-messages). Consequently, messages sent to/from 
+the service should be in this format. Specifically, such messages contain two (2) properties: `header` and `message`,
+where the `header` property contains the Camel headers, and the `message` property contains Camel message.
+The associated Vantiq _schema_ is defined via the `com.vantiq.extsrc.camelconn.camelConnector` assembly imported as 
+described above.  This imported assembly is a prerequisite for the other Vantiq Camel assemblies.
+
+**Note**: Some Vantiq Camel assemblies (specifically those that work with systems with limited messaging
```
Comment (jcguion) [inline camelAssemblies/README.md:129]: Maybe, `For some Vantiq Camel...`  (vs. `Some Vantiq Camel...`)

**Review by jcguion — APPROVED** [review]
LGTM

**Review by AlexBlumer — APPROVED** [review]
LGTM, found one more typo

**Review by rblumer4 — APPROVED** [review]
LGTM   with a few typos

File: camelAssemblies/README.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+This document outlines the contents of the `CamelAssemblies` subproject within the extension sources repo.
+
+The `CamelAssemblies` subproject creates Vantiq projects from
+[Camel Kamelet Definitions](https://camel.apache.org/camel-kamelets/3.21.x/). These projects will
+be made available in a Vantiq Public Catalog as _Vantiq assemblies_. These assemblies, when imported,
+create Vantiq services and sources for connections to or from external systems supported by Apache Camel.
+
+# Prerequisites <a name="pre" id="pre"></a>
+
+**IMPORTANT:** A basic understanding of the [Vantiq Camel Connector](../camelConnector/README.md) is assumed.
```
[the remark this answers is not in the document]
Comment (fhcarter) [reply camelAssemblies/README.md:12]: Added bookmark/tag for #serverConfig. I prefer this over relying on title wording.  But the link nows works.  Thx

**Pull request merged**